assets/less/multilingual.less

```diff
@@ -24,7 +24,7 @@
     }
 
     &.field-multilingual-text {
-        .form-control {
+        input.form-control {
```

Comment:
@mjauvin `form-control` elements aren't always inputs (eg. textareas), so I think it was set to just `.form-control` for that reason. From your Slack convo, you are wanting to fix just the elements in the "fancy" layout, so I would instead add a new block within here somewhere: https://github.com/rainlab/translate-plugin/blob/1b1792caebd0f2904f666b8c2ece0c3f3c6bebc7/assets/less/multilingual.less#L108

Reply:
Whoops, I see that the `.field-multilingual-text` class above it would only apply to text fields. Nonetheless, it's probably better to make the change within the `fancy-layout` block.

Reply:
I'll make the change, makes sense.

Reply:
could not make this change since fancy layout is not used for every multilang text fields.

Reply:
@mjauvin Does the same issue occur with text fields not within a fancy layout though?

Reply:
@bennothommo you're right, it doesn't with the initial CSS. All good now!